ci-operator/step-registry/configure-registry-storage/destroy-s3-cloudfront/configure-registry-storage-destroy-s3-cloudfront-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -f "${SHARED_DIR}/create_aws_bucket.tf" ]]; then
+  echo "Destroy s3 cloudfront setting"
+  cp "${SHARED_DIR}/create_aws_bucket.tf" /tmp && cd /tmp
```

Comment:
Is the original tf file required while removing resources?

Reply:
No need keep it any more, I will use `mv` cmd